identifiers/country-ke/counties.csv

```diff
@@ -0,0 +1,48 @@
+id,name
+ocd-division/country:ke/county:ke-01,Mombasa

```

Comment:
We should drop the `ke-` prefix. It's not included in other countries where we use ISO 3166-2 codes

Reply:
+1 for drop ke-

Reply:
it looks like we still have a `ke` prefix on the county identifiers. I'd recommend dropping those so that the identifiers are just the numbers from the ISO 3166-2 codes.